### PR TITLE
Related to issue #17

### PR DESCRIPTION
**Pull request by frank-wei**
Please review this PR. It helps to add some missing constants used in MaskRCNN.

### CODE REVIEW

File: mlperf_logging/mllog/constants.py [hunk 1]
```diff
@@ -103,9 +103,13 @@
 OPT_LR_WARMUP_FACTOR = "opt_learning_rate_warmup_factor"
 OPT_LR_WARMUP_STEPS = "opt_learning_rate_warmup_steps"
 OPT_WEIGHT_DECAY = "opt_weight_decay"
+MIN_IMAGE_SIZE = "minimum image size"
```
Comment (xyhuang) [inline mlperf_logging/mllog/constants.py:106]: the value after "=" needs to match that in the spec doc (its not a description but actually being printed), e.g. this should be MIN_IMAGE_SIZE = "min_image_size"

File: mlperf_logging/mllog/constants.py [hunk 2]
```diff
@@ -103,9 +103,13 @@
...
+MAX_IMAGE_SIZE = "maximum image size"
```
Comment (xyhuang) [inline mlperf_logging/mllog/constants.py:107]: ditto.

File: mlperf_logging/mllog/constants.py [hunk 3]
```diff
@@ -103,9 +103,13 @@
...
 
 # Log metadata keys
 EPOCH_COUNT = "epoch_count"
 EPOCH_NUM = "epoch_num"
 FIRST_EPOCH_NUM = "first_epoch_num"
 STATUS = "status"
+BBOX = "bbox mAP"
```
Comment (xyhuang) [inline mlperf_logging/mllog/constants.py:114]: ditto.

File: mlperf_logging/mllog/constants.py [hunk 4]
```diff
@@ -103,9 +103,13 @@
...
+SEGM = "segm mAP"
```
Comment (xyhuang) [inline mlperf_logging/mllog/constants.py:115]: ditto.

File: mlperf_logging/mllog/constants.py (hunk 1 above)
Comment (frank-wei) [reply mlperf_logging/mllog/constants.py]: fixed as suggested. Thanks.

File: mlperf_logging/mllog/constants.py (hunk 3 above)
Comment (frank-wei) [reply mlperf_logging/mllog/constants.py]: fixed.

File: mlperf_logging/mllog/constants.py (hunk 2 above)
Comment (frank-wei) [reply mlperf_logging/mllog/constants.py]: fixed

**Pull request merged**